packages/dev/core/src/Particles/computeShaderParticleSystem.ts

```diff
@@ -32,6 +32,12 @@ export class ComputeShaderParticleSystem implements IGPUParticleSystemPlatform {
         this._engine = engine;
     }
 
+    public contextLost(): void {
+        this._updateComputeShader = undefined as any;
+        this._bufferComputeShader = [];
+        this._renderVertexBuffers = [];

```

Comment:
nit - using `.length = 0` will prevent GC and maintain the same object

Reply:
👍 (though I'm not sure that GC is the biggest concerne when there's a context lost ;))